rss2email/feeds.py

```diff
@@ -337,6 +337,9 @@ def _upgrade_state_data(self, data):
             'cannot convert data file from version {} to {}'.format(
                 version, self.datafile_version))
 
+    def configfile_exists(self):
+        return _os.path.exists(self.configfiles[-1])
```

Comment:
This is a reasonably straightforward one-liner, so I don't see the point to wrapping it in a method.  Other than that, this patch looks good to me :).